src/store/reducers/profileReducer.js

```diff
@@ -0,0 +1,83 @@
+import {
+  GET_PROFILE_START,
+  GET_PROFILE_SUCCESS,
+  GET_PROFILE_FAILURE,
+  POST_APPLICANTGRANTS_START,
+  POST_APPLICANTGRANTS_SUCCESS,
+  POST_APPLICANTGRANTS_FAILURE,
+  GET_APPLICANTGRANTS_START,
+  GET_APPLICANTGRANTS_SUCCESS,
+  GET_APPLICANTGRANTS_FAILURE,
+} from "../actions/profileActions";
+
+const initialState = {
+  grants: [],
+  applicantProfileDetails: {},
```

Comment:
This object should be "profileDetails"

Reply:
It was changed later in this pull request to profileDetails